math/finding_number_of_digits_in_a_number.cpp

```diff
@@ -40,6 +42,29 @@ uint64_t finding_number_of_digits_in_a_number(uint64_t n) {
     return count;
 }
 
+/**
+ * @brief This function finds the number of digits
+ * in constant time
```

Comment:
What's the difference between this function and the other function?

Reply:
The second implementation function is using library function floor.

Reply:
@Panquesito7 TCs are different